public/app/plugins/panel/nodeGraph/Node.tsx

```diff
@@ -86,13 +88,20 @@ export const Node = memo(function Node(props: {
   const isHovered = hovering === 'active';
   const nodeRadius = node.nodeRadius?.values[node.dataFrameRowIndex] || nodeR;
   const strokeWidth = computeNodeCircumferenceStrokeWidth(nodeRadius);
+  const fontSize = (node.fontSize?.values[node.dataFrameRowIndex] || fontS) + 'px';
```

Comment:
Should we do any runtime validation with a fallback that the dataFrame contains a numeric and valid fontSize?